extensions/guacamole-auth-duo/pom.xml

```diff
@@ -209,19 +209,11 @@
 
     <dependencies>
 
-        <!-- Guacamole Java API -->
-        <dependency>
```

Comment:
Similar to apache/incubator-guacamole-manual#23, I removed the explicit dependencies on guacamole-common to rely instead on the transitive dependency pulled via guacamole-ext.

Think that's a bad idea/practice?